pkg/lightning/restore/restore.go

```diff
@@ -1798,26 +1760,58 @@ func (rc *Controller) isLocalBackend() bool {
 // 3. Lightning configuration
 // before restore tables start.
 func (rc *Controller) preCheckRequirements(ctx context.Context) error {
-	if !rc.cfg.App.CheckRequirements {
-		log.L().Info("skip pre check due to user requirement")
-		return nil
-	}
```

Comment:
removing this means all the pre-check are now always run, but all failures are just hidden away. why is this changed 🤔

Reply:
Because some tests shall be  always checked.